README.md

```diff
@@ -6,7 +6,7 @@ The main credentials, **databaseURL** & the **storageBucket** are automatically
 # each arg must have at least 2-part key (e.g foo.bar)
 $ firebase functions:config:set send_in_blue.key="sendInBlue secret Key"
 # Coordinator details
-$ firebase functions:config:set sqr.allotment.templateid=NUMBER
+$ firebase functions:config:set sqr.allotment.templateid='String | template name'
```

Comment:
This is not needed anymore as we will identify templates by name.

Reply:
it's still needed in sending mail to an assignee on a anew allotment...
it's not specified in the **SQR Allotments** data strcuture.